docs/DeveloperGuide.md

```diff
@@ -625,13 +625,17 @@ The `EditEventCommand` class is responsible for editing a specific Event for a p
 
 #### Specifications
 
-* EditEventCommand takes in four parameters: `PATIENT_INDEX`, `EVENT_INDEX`, `EVENT_NAME` and `DATE`.
-  All parameters are compulsory. Optionally, you may include `TIME` in the `DATE` parameter.
+* EditEventCommand takes in four parameters: `PATIENT_INDEX`, `EVENT_INDEX`, `NAME_OF_EVENT` 
```

Comment:
Perhaps give example of with and without the `TIME`?

Reply:
okay! Will do it in my next PR since this PR is merge! Thanks for the suggestion:)